README.md

```diff
@@ -240,132 +291,90 @@ const invoices = {
 const contentType = mime.lookup(filename);
 
-const accountAttachmentsResponse: any = await xero.accountingApi.createAccountAttachmentByFileName(req.session.activeTenant.tenantId, accountId, filename, readStream, {
+const accountAttachmentsResponse = await xero.accountingApi.createInvoiceAttachmentByFileName(activeTenantId, invoiceId, filename, readStream, {
   headers: {
     'Content-Type': contentType
   }
 });
+```
 
-// UPLOAD A FILE
-import * as fs from "fs";
-const mime = require("mime-types");
-const path = require("path");
+---
+## SDK conventions
 
-const tenantId = 'valid-xero-tenant-id-uuid'
-const folderId = 'valid-folder-uuid-goes-here'
 
-const filename = "xero-dev.png";
-const pathToUpload = path.resolve(__dirname, "../public/images/xero-dev.png");
-const readStream = fs.createReadStream(pathToUpload);
-const contentType = mime.lookup(filename);
+### Querying & Filtering
 
-const uploadFile = await xero.filesApi.uploadFile(tenantId, folderId, readStream, filename, contentType);
-```
+Descibe the support for query options and filtering
 
-# Querying With Filters And Pagination
 ```js
-const xeroTenantId = 'YOUR_XERO_TENANT_ID';
+const activeTenantId = 'XERO_TENANT_ID';
 const ifModifiedSince: Date = new Date("2020-02-06T12:17:43.202-08:00");
 const where = 'Status=="AUTHORISED" AND Type=="SPEND"';
 const order = 'Reference ASC';
 const page = 1;
 const unitdp = 4;
 
-const response = await xero.accountingApi.getBankTransactions(xeroTenantId, ifModifiedSince, where, order, page, unitdp);
+const response = await xero.accountingApi.getBankTransactions(activeTenantId, ifModifiedSince, where, order, page, unitdp);
 ```
-
-# Security
-This repo leverages a certified OA2 and OIDC library called openid-client. For a deeper dive the repo's functionality, check out them directly https://github.com/panva/node-openid-client.
-
-### Preventing CSRF Using Xero-Node
-When xero.buildConsentUrl is called we call openid-client authorizationUrl method, passing redirect_uri, scope, and state (if present) as arguments and returns a formatted url string made up from the given config. The user is then directed to the consentUrl to begin the auth process with Xero. When the auth process is complete Xero redirects the user to the specified callback route and passes along params including the state if it was initially provided.
-
-At this point openid-client takes over verifying params.state and check.state match if provided. If the state does not match the initial user's, the openid-client library throws an error:
-
-```json
-RPError: state mismatch, expected user=1234, got: user=666
-```
-###  JWT Verification Using Xero-Node
-JWT verification of both the `access_token` and `id_token` are baked into the openid-client library we leverage.
-
-When `xero.apiCallback` is called, openid-client `validateJARM` is triggered which also invokes `validateJWT`
-
-If openid-client fails to validate the JWT signature it will throw an error. 
-
```

Comment:
Do you think we want to get rid of these sections?

### Security
### Preventing CSRF Using Xero-Node
### JWT Verification Using Xero-Node

Reply:
Good catch! adding back now